src/Blink.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace DoSomething\Gateway;
+
+use DoSomething\Gateway\Common\RestApiClient;
+use DoSomething\Gateway\Resources\NorthstarUser;
+
+class Blink extends RestApiClient
+{
+    use AuthorizesWithBlink;
+
+    /**
+     * Configuration array.
+     *
+     * @var string
+     */
+    protected $config;
+
+    /**
+     * Default headers applied to every request.
+     *
+     * @var array
+     */
+    protected $defaultHeaders;
+
+    /**
+     * Create a new Blink API client.
+     * @param array $config
+     * @param array $overrides
+     */
+    public function __construct($config = [], $overrides = [])
+    {
+        // Save configuration.
+        $this->config = $config;
+
+        // Set response header.
+        $this->auth = [];
+        if (! empty($config['user'])) {
+            $this->auth['user'] = $config['user'];
+        }
+        if (! empty($config['password'])) {
+            $this->auth['password'] = $config['password'];
+        }
+        parent::__construct($config['url'], $overrides);
+    }
+
+    /**
+     * Send a Post request Blink /events/user-create endpoint.
+     *
+     * To notify Blink that user has been created.
+     *
+     * @param string $id - Signup
+     * @return bool
+     */
+    public function userCreate(NorthstarUser $user)
```

Comment:
Let's allow any array here, since one of the major use-cases for this is serializing a Eloquent model in Northstar itself (rather than a response provided via Gateway).